extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -227,6 +229,19 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     return true;
   }
 
+  /**
+   * Set sidekick ads
+   */
+  maybeSetStyleForSticky_() {
+    if (this.element.hasAttribute('sticky')) {
```

Comment:
Is this whitelisted in validator?

Reply:
Yes https://github.com/ampproject/amphtml/pull/29659